terraform/cloud-platform-components/nginx-ingress-acme.tf

```diff
@@ -59,6 +59,7 @@ data "template_file" "nginx_ingress_default_certificate" {
 
   vars {
     common_name = "*.apps.${data.terraform_remote_state.cluster.cluster_domain_name}"
+    alt_name    = "${terraform.workspace == local.live_workspace ? format("- *.%s", local.live_domain) : ""}"
```

Comment:
Is that `-` correct?

Reply:
yes, sadly, indenting :(

Reply:
I think you missed committing changes to the cert template?